files/deployment.yaml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Build packit-service image form current src dir.
+  hosts: all
+  vars:
+    deployment_dir: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/packit-service/deployment'].src_dir }}"
+  tasks:
+  - name: Install packages for deployment
+    dnf:
+      name:
+      - ansible-2.7.10-1.fc30 # because of packit-service deployment
```

Comment:
I fear this will stop working once that build stops being in a compose. I guess we can solve it once that happens.